libraries/dotnet/EnergyOrigin.TokenValidation/doc/auth-subsystem/token-validation.md

```diff
@@ -0,0 +1,147 @@
+# Token Validation
+
+Token validation is built around facilitating the [Security and Identity](https://learn.microsoft.com/en-us/aspnet/core/security/authorization/introduction?view=aspnetcore-7.0) concept in ASP.NET Core.
+
+For our specific use cases, see [restricting access](#restricting-access).
+
+## Quick Start
+
+1. Register token validation during start up:
+
+```csharp
+var options = new TokenValidationOptions {
```

Comment:
Why not just delete all Token Validation stuff, since it's not needed anymore.

Reply:
When Old auth is gone, we can do so.